### ISSUE TITLE
1.19.3 compatibility

### ISSUE THREAD

**Issue by SlimeDog** [Zrips/Residence#970]
Spigot 1.19.3-3616
CMILib 1.2.3.7
Residence 5.1.0.1

On server start-up
```
[07:27:42] [Server thread/INFO]: [Residence] Enabling Residence v5.1.0.1
[07:27:42] [Server thread/INFO]: [Residence] Disabling Residence v5.1.0.1
[07:27:42] [Server thread/INFO]: ^[[0;32;1m[^[[0;33;22mResidence^[[0;32;1m]^[[0;37;22m Loaded (2) groups^[[m
[07:27:42] [Server thread/INFO]: ^[[0;32;1m[^[[0;33;22mResidence^[[0;32;1m]^[[0;37;22m Found LuckPerms5 Plugin!^[[m
[07:27:42] [Server thread/INFO]: ^[[0;32;1m[^[[0;33;22mResidence^[[0;32;1m]^[[0;37;22m Found WorldEdit 7.2.13-SNAPSHOT+6282-2e86f05^[[m
[07:27:42] [Server thread/INFO]: ^[[0;32;1m[^[[0;33;22mResidence^[[0;32;1m]^[[0;37;22m - FAILED INITIALIZATION! DISABLED! ERROR:^[[m
[07:27:42] [Server thread/ERROR]: null
java.lang.NullPointerException: Cannot invoke "org.bukkit.inventory.ItemStack.getType()" because "this.item" is null
        at net.Zrips.CMILib.Items.CMIItemStack.getItemStack(CMIItemStack.java:387) ~[CMILib-1.2.3.7.jar:?]
        at com.bekvon.bukkit.residence.gui.FlagUtil.load(FlagUtil.java:68) ~[?:?]
        at com.bekvon.bukkit.residence.Residence.onEnable(Residence.java:494) ~[?:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:264) ~[spigot-api-1.19.3-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:342) ~[spigot-api-1.19.3-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:480) ~[spigot-api-1.19.3-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.craftbukkit.v1_19_R2.CraftServer.enablePlugin(CraftServer.java:539) ~[spigot-1.19.3-R0.1-SNAPSHOT.jar:3616-Spigot-d90018e-eecb4c0]
        at org.bukkit.craftbukkit.v1_19_R2.CraftServer.enablePlugins(CraftServer.java:453) ~[spigot-1.19.3-R0.1-SNAPSHOT.jar:3616-Spigot-d90018e-eecb4c0]
        at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:587) ~[spigot-1.19.3-R0.1-SNAPSHOT.jar:3616-Spigot-d90018e-eecb4c0]
        at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:412) ~[spigot-1.19.3-R0.1-SNAPSHOT.jar:3616-Spigot-d90018e-eecb4c0]
        at net.minecraft.server.dedicated.DedicatedServer.e(DedicatedServer.java:250) ~[spigot-1.19.3-R0.1-SNAPSHOT.jar:3616-Spigot-d90018e-eecb4c0]
        at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:973) ~[spigot-1.19.3-R0.1-SNAPSHOT.jar:3616-Spigot-d90018e-eecb4c0]
        at net.minecraft.server.MinecraftServer.lambda$0(MinecraftServer.java:301) ~[spigot-1.19.3-R0.1-SNAPSHOT.jar:3616-Spigot-d90018e-eecb4c0]
        at java.lang.Thread.run(Thread.java:1589) ~[?:?]
[07:27:42] [Server thread/ERROR]: Error occurred while enabling Residence v5.1.0.1 (Is it up to date?)
java.lang.IllegalStateException: zip file closed
        at java.util.zip.ZipFile.ensureOpen(ZipFile.java:832) ~[?:?]
        at java.util.zip.ZipFile.getEntry(ZipFile.java:330) ~[?:?]
        at java.util.jar.JarFile.getEntry(JarFile.java:516) ~[?:?]
        at java.util.jar.JarFile.getJarEntry(JarFile.java:471) ~[?:?]
        at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:159) ~[spigot-api-1.19.3-R0.1-SNAPSHOT.jar:?]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:588) ~[?:?]
        at org.bukkit.plugin.java.PluginClassLoader.loadClass0(PluginClassLoader.java:104) ~[spigot-api-1.19.3-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.java.PluginClassLoader.loadClass(PluginClassLoader.java:99) ~[spigot-api-1.19.3-R0.1-SNAPSHOT.jar:?]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:521) ~[?:?]
        at com.bekvon.bukkit.residence.Residence.getShopSignUtilManager(Residence.java:1003) ~[?:?]
        at com.bekvon.bukkit.residence.Residence.onEnable(Residence.java:842) ~[?:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:264) ~[spigot-api-1.19.3-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:342) ~[spigot-api-1.19.3-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:480) ~[spigot-api-1.19.3-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.craftbukkit.v1_19_R2.CraftServer.enablePlugin(CraftServer.java:539) ~[spigot-1.19.3-R0.1-SNAPSHOT.jar:3616-Spigot-d90018e-eecb4c0]
        at org.bukkit.craftbukkit.v1_19_R2.CraftServer.enablePlugins(CraftServer.java:453) ~[spigot-1.19.3-R0.1-SNAPSHOT.jar:3616-Spigot-d90018e-eecb4c0]
        at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:587) ~[spigot-1.19.3-R0.1-SNAPSHOT.jar:3616-Spigot-d90018e-eecb4c0]
        at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:412) ~[spigot-1.19.3-R0.1-SNAPSHOT.jar:3616-Spigot-d90018e-eecb4c0]
        at net.minecraft.server.dedicated.DedicatedServer.e(DedicatedServer.java:250) ~[spigot-1.19.3-R0.1-SNAPSHOT.jar:3616-Spigot-d90018e-eecb4c0]
        at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:973) ~[spigot-1.19.3-R0.1-SNAPSHOT.jar:3616-Spigot-d90018e-eecb4c0]
        at net.minecraft.server.MinecraftServer.lambda$0(MinecraftServer.java:301) ~[spigot-1.19.3-R0.1-SNAPSHOT.jar:3616-Spigot-d90018e-eecb4c0]
        at java.lang.Thread.run(Thread.java:1589) ~[?:?]
```

**Comment by PetyXbron** [comment]
I have the same error here! According to [the latest CMIlib commit](https://github.com/Zrips/CMILib/commit/2a6a954e90e845f6eab9a3f567362724f0d8a4f2) CMIlib supports (already?) 1.19.3 but I could not find the .jar file. Also, I am not sure if Residence has 1.19.3 support directly (but it just should have CMIlib updated)

@Zrips can you please check this issue? **This error shut downs the server so we can't use the enabled Residence plugin**.
Also where we can get the latest CMIlib .jar file if it's not on [the Spigot page](https://www.spigotmc.org/resources/cmilib.87610/)?

**Comment by Zrips** [comment]
1.19.3 support comes out this week. As sites like paper still marks any 1.19.3 builds as experimental, you should not use it for live server at this time, give it a few weeks before major bugs gets ironed out. 
I did updated CMILib github page with initial 1.19.3 support, but for final version I will still need to do extra work on it. Give me a few days for that.

**Comment by Zrips** [comment]
https://www.zrips.net/CMILib/CMILib1.2.4.0-dev.jar
If you really want to mess around with it, but it will have issues, mainly relating to NBT data handling, which I will fix monday most likely. This will allow you to run most of the plugins, atleast start up server

**Comment by SlimeDog** [comment]
This morning, PaperMC released "stable" paper-1.19.3-320.
